titus-api/src/main/java/com/netflix/titus/api/FeatureRolloutPlans.java

```diff
@@ -63,25 +63,11 @@
     )
     String JOB_AUTHORIZATION_FEATURE = "jobAuthorizationFeature";
 
-    @FeatureRollout(
```

Comment:
I didn't see a removal of this in the code. Was it ever used?

Reply:
I think the use got removed on a previous PR that removed other JAH pieces